awx/main/tests/unit/utils/test_ansible.py

```diff
@@ -0,0 +1,33 @@
+import os
+import os.path
+
+import pytest
+
+from awx.main.tests import data
+from awx.main.utils.ansible import could_be_playbook, could_be_inventory
+
+DATA = os.path.join(os.path.dirname(data.__file__), 'ansible_utils')
```

Comment:
that's much more clever than backing up through parent dirs with `..` over and over